code/modules/mob/living/simple_animal/hostile/grinch.dm

```diff
@@ -7,10 +7,24 @@
 	icon_dead = "grinch_dead"
 	held_items = list(null, null)
 	mob_bump_flag = PASSTABLE // Pass over everything
+	mutations = list(M_CLUMSY)
```

Comment:
[[ARE YOU SURE YOU WANT TO DO THIS?]]

Reply:
They're supposed to be unable to use guns

Reply:
Doesn't M_CLUMSY do a little more than just disable gun use comedically?

Reply:
Guns, cuffs, various instruments of murderbone.

Reply:
Clumsy means grinches are affected by stuff like clowns are, which includes cuffing, shooting, e-swords, and more. Would be more convenient to just give them a message and a return 0 when they try to fire weapons

Reply:
I see no problem with making Grinches equivalent to clown in murderbone capacity